packages/contextual-help/test/contextual-help.test.ts

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import {
+    elementUpdated,
+    expect,
+    fixture,
+    html,
+    oneEvent,
+} from '@open-wc/testing';
+import { nextFrame } from '@spectrum-web-components/overlay/src/AbstractOverlay.js';
+import { sendKeys } from '@web/test-runner-commands';
+import { testForLitDevWarnings } from '../../../test/testing-helpers.js';
+import { ContextualHelp } from '../src/ContextualHelp.js';
+import '../sp-contextual-help.js';
+
+describe('ContextualHelp', () => {
+    testForLitDevWarnings(
+        async () =>
+            await fixture<ContextualHelp>(
+                html`
+                    <sp-contextual-help headline="Permission required">
+                        Your admin must grant you permission before you can
+                        create a segment.
+                    </sp-contextual-help>
+                `
+            )
```

Comment:
Once you have a `template.ts` file, it would be great to generate the test context from there.